requirements.txt

```diff
@@ -1,44 +1,44 @@
 # Autogenerated by 'make bs'
 -i https://pypi.org/simple
-bracex==2.3.post1; python_version >= '3.7'
-cairocffi==1.3.0; python_version >= '3.7'
+bracex==2.3.post1 ; python_version >= '3.7'
```

Comment:
Do you use devbox or Mac? I get these same whitespace diffs (on devbox), been wondering if it's platform-dependent.

Reply:
I'm on mac